src/open_prime_rando/dol_patching/echoes/dol_patches.py

```diff
@@ -500,3 +500,50 @@ def apply_map_door_changes(door_symbols: MapDoorTypeAddresses, dol_file: DolFile
 
     dol_file.symbols["CTweakAutoMapper::GetDoorColor::DoorColorArray"] = door_color_array
     dol_file.write("CTweakAutoMapper::GetDoorColor::DoorColorArray", DoorMapIcon.get_surface_colors_as_bytes())
+
+
+def apply_widescreen_hack(version_description: str, dol_file: DolFile, enabled: bool):
+    """
+    Apply widescreen render hack to render the game in 16:9
+    """
+    # Ported from gamemasterplc's 16:9 gecko code for NTSC-U
+    match version_description:
+        case "Gamecube NTSC":
+            culling_replacement = 0x8030256C
+            culling_insertion = 0x80418E8C
+            viewport_replacement = 0x8036D684
+            viewport_insertion = 0x80003748
```

Comment:
these should be new fields added to `EchoesDolVersion` rather than a custom implementation like this. see other functions in this file for reference

Reply:
Moved these addresses to dol_versions.py